composed_api.rst

```diff
@@ -287,20 +276,20 @@ Endpoint
 
 ::
 
-    GET <instance_url>/<instance_root>_ba/api/v1/aoi/add
+    GET <instance_url>/<instance_root>_ba/api/v2/aoi/add
 
 Request Parameters
 ^^^^^^^^^^^^^^^^^^
 
 +-------------------+-------------------------------------------------------+
 | Query parameter   | Value                                                 |
 +===================+=======================================================+
+| source            | *Required*. Your GRiD generated API key.              |
```

Comment:
You should be able to build GQ using a single key you generated on a GRiD system (running 1.4.2).  The end user shouldn't need to get their own.  If GQ requires access to multiple systems, then you will need to generate that key using the same name on each GRiD system that it needs access.  The key generated is essentially a hash derived from the name to provide consistency across instances.

Does that sound closer to expectations to you?  Definitely let us know if you think there are better ways to improve this.